pubsub/pubsub.go

```diff
@@ -51,8 +53,75 @@ type Client struct {
 	subc      *vkit.SubscriberClient
 }
 
-// NewClient creates a new PubSub client.
+// ClientConfig has configurations for the client.
+type ClientConfig struct {
+	PublisherCallOptions  *vkit.PublisherCallOptions
+	SubscriberCallOptions *vkit.SubscriberCallOptions
+}
+
+// mergePublisherCallOptions merges two PublisherCallOptions into one and the first argument has
+// a lower order of precedence than the second one. If either is nil, return the other.
+func mergePublisherCallOptions(a *vkit.PublisherCallOptions, b *vkit.PublisherCallOptions) *vkit.PublisherCallOptions {
```

Comment:
I think you could write a generic "merge two structs of the same type" function, but this works fine. 👍